docs/eventing/sources/rabbitmq-source/README.md

```diff
@@ -0,0 +1,219 @@
+# Creating a RabbitMQ Source
+
+This topic describes how to create a RabbitMQ Source.
+
+## Prerequisites
+
+To use the RabbitMQ Source, you must have the following installed:
+
+1. [Knative Eventing](../../../install/yaml-install/eventing/install-eventing-with-yaml.md)
+1. [RabbitMQ Cluster Operator (optional)](https://github.com/rabbitmq/cluster-operator) - our recommendation is [latest release](https://github.com/rabbitmq/cluster-operator/releases/latest)
```

Comment:
it says must have above, but then this link says optional - I'd rethink this prereq section